secboot/tpm.go

```diff
@@ -197,6 +205,16 @@ func (t *tpmSupport) Seal(key []byte, keyPath, policyUpdatePath string) error {
 		return fmt.Errorf("cannot add systemd EFI stub profile: %v", err)
 	}
 
+	// Add snap model profile
```

Comment:
should we do this bit only if len(t.models) != 0 ?   SnapModelProfilesParams will error if Models is empty

Reply:
Yes, in this case I think we should do it conditionally.